guides/common/modules/proc_creating-a-custom-ssl-certificate.adoc

```diff
@@ -11,6 +11,15 @@ On {ProjectServer}, create a custom certificate for your {ProductName}.
 If you already have a custom SSL certificate for {ProductName}, skip this procedure.
 endif::[]
 
+ifeval::["{context}" == "load-balancing-with-custom-ssl-certificates-with-puppet"]
+:context: load-balancing
+endif::[]
```

Comment:
I'm not sure if changing the context mid-air is a good idea.
Perhaps you can use `ifdef:load-balancing[]` instead of ifeval.
This attribute is set for the LB guide.

Reply:
So I have to manipulate the context because there are two instances of `common/modules/proc_creating-a-custom-ssl-certificate.adoc` in `doc-Configuring_Load_Balancer/master.adoc`, this fails because there are duplicate IDs in the same doc.

Reply:
So perhaps instead of using context to populate the values in the modules you can use a new attribute and set it on the `master.adoc` level? Then you don't have to change the context.

Reply:
Introduced a new attribute `context_title`.